lib/spear/connection/request.ex

```diff
@@ -259,6 +260,21 @@ defmodule Spear.Connection.Request do
         request = put_in(request.from, nil)
         put_in(request.response.data, rest)
 
+      {Monitoring.stats_resp() = message, rest} ->
+        request =
+          update_in(request.from, fn
+            nil ->
+              nil
+
+            from ->
+              GenServer.reply(from, {:ok, request.request_ref})
+
+              nil
+          end)
+
+        send(subscriber, through.(message, request.request_ref))
+        put_in(request.response.data, rest)
```

Comment:
subscribe_to_stats/3 is a bit weird because it's a subscription like subscribe/4 or connect_to_persistent_subscription/5 but there's no confirmation message, it just starts emitting stats, so it doesn't look like the clauses above

this is a bit hacky but I don't really mind, optimizing this route seems a little frivolous